web/src/containers/Admin/Broker/index.js

```diff
@@ -6,7 +6,8 @@ import BrokerForm from './BrokerForm';
 import { getConstants, updatePlugins } from '../Plugins/action';
 import { getFees } from '../Fees/actions';
 import { validateRequired } from '../../../components/AdminForm/validations';
-import { formatCurrency } from '../../../utils/currency';
+import { formatCurrencyByIncrementalUnit } from '../../../utils/currency';
```

Comment:
I know that we already have many relative import leftovers in the codebase from the past. But it's better to use absolute import for better readability and easy refactoring.